kclvm/runtime/src/value/api.rs

```diff
@@ -652,9 +659,6 @@ pub unsafe extern "C" fn kclvm_value_function_invoke(
                 args_new.list_append_unpack(&closure_new);
                 call_fn(ctx, args_new.into_raw(), kwargs)
             // Normal kcl function, call directly
-            } else if is_external {
```

Comment:
Why did you delete here?

Reply:
The previous code simply checked whether the external name is empty, and if it is empty, the plugin will be called. However, in order to display the backtrace, all function calls need to be given their external name, so the plugin related code is temporarily removed.

Reply:
> The previous code simply checked whether the external name is empty, and if it is empty, the plugin will be called. However, in order to display the backtrace, all function calls need to be given their external name, so the plugin related code is temporarily removed.

OK. But.. In fact, we did not completely remove the kcl plugin agent function in version 0.4.6, and hastily deleting it would result in user-side incompatibility, so it can be retained for now.